lib/instance-validator.js

```diff
@@ -351,24 +369,40 @@ class InstanceValidator {
   _handleReflectedResult(field, value, promiseInspections) {
     for (const promiseInspection of promiseInspections) {
       if (promiseInspection.isRejected()) {
-        const rejection = promiseInspection.error();
-        this._pushError(true, field, rejection, value);
+        const
+          rejection = promiseInspection.error(),
+          isBuiltIn = !!rejection.validatorName;
```

Comment:
there is no rule about this in your `.eslintrc`, and no other coding style documentation exists (any more?) ... I'm a bit loath to start making CS changes based on one person's POV ... what happens when janmeier asks for the exact opposite? :-) ... I've changed it but I think it this is a CS rule it really should be in the `.eslintrc` (and if something lints correctly then I feel it should be let through, with regard to CS at least) 

I generally write my consts like this because of a vague understanding that V8 prefers it this way.